src/systemctl/systemctl.c

```diff
@@ -5668,6 +5713,12 @@ static int enable_unit(int argc, char *argv[], void *userdata) {
 
                 polkit_agent_open_if_enabled();
 
+                if (STR_IN_SET(verb, "mask", "unmask")) {
+                        r = unit_exists(*names, verb);
+                        if (r != 0)
+                                return r;
```

Comment:
For example if you are going to install a package and want to prevent the unit from being started. Or for example if you want to mask a bunch of units, without really caring if they are all installed atm.